README.md

```diff
@@ -1,6 +1,10 @@
 # RTB House
 
-Open the VTEX APP Store and install the app on your store.
+The RTB House engine, one of its kind powered by **deep learning algorithms**, identifies potential buyers and boosts performance via 1:1 ultra-personalized retargeting **up to 50 percent more efficient** than standard machine learning AI-based approaches.
+For more information please visit, [RTB HOUSE](https://www.rtbhouse.com/) website.
+
+## App Installation
+Open the VTEX APP Store and install the RTB HOUSE app on your store.
 
 or
 
```

Comment:
You dont need the root readme file

Reply:
readme file removed from root.
thanks Savio!